apps/studio/components/interfaces/Auth/EmailTemplates/TemplateEditor.tsx

```diff
@@ -284,76 +288,80 @@ export const TemplateEditor = ({ template }: TemplateEditorProps) => {
-                                onClick={() => insertTextAtCursor(variable)}
-                              >
-                                {variable}
-                              </Button>
-                            </TooltipTrigger>
-                            <TooltipContent side="top">
-                              <p>{description || 'Variable description not available'}</p>
-                            </TooltipContent>
-                          </Tooltip>
-                        ))}
-                      </div>
+                    <div className="flex flex-wrap gap-1">
+                      {messageVariables.map(({ variable, description }) => (
+                        <Tooltip key={variable}>
+                          <TooltipTrigger asChild>
+                            <Button
+                              type="outline"
+                              size="tiny"
+                              className="rounded-full"
+                              onClick={() => insertTextAtCursor(variable)}
+                            >
+                              {variable}
+                            </Button>
+                          </TooltipTrigger>
+                          <TooltipContent side="top">
+                            <p>{description || 'Variable description not available'}</p>
+                          </TooltipContent>
+                        </Tooltip>
+                      ))}
                     </div>
                   )}
-                </TabsContent_Shadcn_>
-                <TabsContent_Shadcn_ value="preview" className="pt-0 mt-0">
+                </>
+              ) : (
+                <>
                   <iframe
-                    className="!mb-0 mt-0 overflow-hidden h-96 w-full"
+                    className="!mb-0 mt-0 overflow-hidden h-96 w-full rounded-md border bg-white"
                     title={id}
                     srcDoc={bodyValue}
                     sandbox="allow-scripts allow-forms"
                   />
                   <Admonition
                     type="default"
-                    title="The preview may differ slightly from the actual rendering in the email client"
-                    className="rounded-none border-0 mb-0"
+                    title="Email rendering may differ"
+                    description="The preview shown here may differ slightly from how your email appears in the recipient’s email client."
                   />
-                </TabsContent_Shadcn_>
-              </Tabs_Shadcn_>
-            </CardContent>
-            <CardContent>
-              <SpamValidation validationResult={validationResult} />
+                </>
+              )}
             </CardContent>
-            <CardFooter className="justify-end space-x-2">
+
+            {spamRules.length > 0 && (
+              <CardContent>
+                <SpamValidation validationResult={validationResult} />
+              </CardContent>
+            )}
```

Comment:
As mentioned above: only show the SpamValidation component when there are spam issues.